site/docs/pages/guides/writing-messages.mdx

```diff
@@ -85,6 +113,25 @@ and return early if so.
 If you don't have a funded account you can use, note the address and private key pair that's logged. Transfer funds to the address and use the same private key as the `CUSTODY_PRIVATE_KEY` for the next run of the script. 
 
 ```typescript
+// Create wallet and public clients
+const account = privateKeyToAccount(CUSTODY_PRIVATE_KEY as `0x${string}`);
+const accountPrivateKey = CUSTODY_PRIVATE_KEY === '<REQUIRED>' ? generatePrivateKey() : CUSTODY_PRIVATE_KEY;
```

Comment:
This update seems weird? accountPrivateKey should be defined before account and then we should use accountPrivateKey instead of CUSTODY_PRIVATE_KEY to initialize account.